packages/app/src/explorer/Explorer.module.css

```diff
@@ -55,6 +55,7 @@
 }
 
 .name {
+  padding: 0.375rem 0;
```

Comment:
Here and above, I'm fixing a random cropping issue I've noticed in the Explorer:

![image](https://user-images.githubusercontent.com/2936402/151136473-cd04b791-25a8-4caf-b15f-2c60b980c053.png)

This is caused by `overflow: hidden` below, which is required for the ellipsis to appear when the explorer is too narrow. The fix is to move the padding from the parent to the child with the overflow (`.name`).